Makefile

```diff
@@ -6,13 +6,17 @@ init:
 	pip install -e '.[dev]'
 
 test:
-	pytest --cov samtranslator --cov-report term-missing --cov-fail-under 95 tests
+	pytest --cov samtranslator --cov-report term-missing --cov-fail-under 95 tests/*
+
+# Need to specify *.py or pytest attempts to execute the README.md file
+test-integ:
+	pytest --no-cov tests_integ/**/*.py
```

Comment:
`**` is not supported in POSIX `sh`; which shell is used to run this Makefile?

Reply:
At least for [GNU Make](https://www.gnu.org/software/make/manual/html_node/Choosing-the-Shell.html):

> The program used as the shell is taken from the variable SHELL. If this variable is not set in your makefile, the program /bin/sh is used as the shell.

Which suggests this isn't correct.

Why do we need to specify in that way? Instead of `pytest --no-cov tests_integ`.

Reply:
Would tend to concur, we should just be able to run on the folder.

Reply:
Fixed